src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/services/EnvelopeRetrieverServiceTest.java

```diff
@@ -38,7 +41,7 @@ public void setUp() {
     }
 
     @Test
-    public void should_return_all_envelopes_successfully_for_a_given_jurisdiction() throws Exception {
+    public void should_return_all_envelopes_successfully_for_a_given_jurisdiction_and_status() throws Exception {
```

Comment:
We could few one test case i.e when invalid status is passed

Reply:
add 2 more tests here:
https://github.com/hmcts/bulk-scan-processor/pull/101/commits/e038d4ce70b7ac37c5c8a9119c6eabafe39f3ac7